### ISSUE TITLE
Accessibility problems using NVDA with IE 11

### ISSUE THREAD

**Issue by iamjett27** [nvaccess/nvda#5852]
Hello,
I have some accessibility problems using NVDA with IE 11. 
I use Devexpress ASPxButton, I have put a label before each button so that NVDA can read buttons by pressing 'b'. These buttons also have tooltips (for example a Refresh button). The problem is that in IE 11, when I press 'b', NVDA first reads the tooltip with the word 'button' at the end, then it reads 'Refresh" button.
When I removed the tooltip, NVDA read 'Refresh button' and then 'button button'. 
I don't have any of these problems with firefox.

**Comment by jcsteh** [comment]
We really need to see the HTML code that this generates before we can 
diagnose this. Thanks.

**Comment by iamjett27** [comment]
Thanks, please update me as soon as you can.

**Comment by jcsteh** [comment]
Let me rephrase: please post the HTML code that this generates. Without this, we can't do anything further. Thanks.

**Comment by iamjett27** [comment]
Thanks, but how and where do i get the HTML code that you need?

**Comment by jcsteh** [comment]
Your ASP code generates HTML which it provides to the browser. You 
should be able to use web browser developer tools or the like to get the 
HTML code the browser is rendering.

**Comment by iamjett27** [comment]
ok thanks. will do that.

**Comment by iamjett27** [comment]
Here is my original HTML code:
<label for="<%=btnRefresh1.ClientID%>" tabindex="4" role="button" style="outline:none">
   <dx:ASPxButton CssClass="btn floatButtonList dxCenterBugFix" style="margin-top:0px;margin-bottom:15px;" VerticalAlign="Middle" HorizontalAlign="Center" ID="btnRefresh1" runat="server" Image-Url="~/Images/refresh.png" ImagePosition="Top" AutoPostBack="false">
   <ClientSideEvents Click="function(s,e){Refresh();}" />                         
   </dx:ASPxButton>
</label>

and here is the code that is generated to the browser:
<table style="border-collapse: separate;" border="0" cellspacing="0" cellpadding="0">

   <tbody>
      <tr>
                    
         <td style="padding-bottom: 4px;">
            <img id="ctl00_ctl00_ctl00_PageContent_BodyContent_BodyPageContent_btnRefresh1_BImg" style="border-width: 0px;" alt="" src="Images/refresh.png">
         </td>
                
      </tr>
      <tr>
                    
         <td class="dxb" id="ctl00_ctl00_ctl00_PageContent_BodyContent_BodyPageContent_btnRefresh1_BTC">
            <span>Refresh</span>
         </td>
                
      </tr>
            
    </tbody>
</table>

**Comment by iamjett27** [comment]
Here is my original HTML code:
<label for="<%=btnRefresh1.ClientID%>" tabindex="4" role="button" style="outline:none">
   <dx:ASPxButton CssClass="btn floatButtonList dxCenterBugFix" style="margin-top:0px;margin-bottom:15px;" VerticalAlign="Middle" HorizontalAlign="Center" ID="btnRefresh1" runat="server" Image-Url="~/Images/refresh.png" ImagePosition="Top" AutoPostBack="false">
   <ClientSideEvents Click="function(s,e){Refresh();}" />                         
   </dx:ASPxButton>
</label>

and here is the code that is generated to the browser:
<table style="border-collapse: separate;" border="0" cellspacing="0" cellpadding="0">

   <tbody>
      <tr>
                    
         <td style="padding-bottom: 4px;">
            <img id="ctl00_ctl00_ctl00_PageContent_BodyContent_BodyPageContent_btnRefresh1_BImg" style="border-width: 0px;" alt="" src="Images/refresh.png">
         </td>
                
      </tr>
      <tr>
                    
         <td class="dxb" id="ctl00_ctl00_ctl00_PageContent_BodyContent_BodyPageContent_btnRefresh1_BTC">
            <span>Refresh</span>
         </td>
                
      </tr>
            
    </tbody>
</table>

**Comment by iamjett27** [comment]
Here is my original HTML code:
[HTML code.txt](https://github.com/nvaccess/nvda/files/203512/HTML.code.txt)

**Comment by jcsteh** [comment]
The label element seems to be missing in the browser HTML code. I assume this is an error and it wraps the table element?

Applying role="button" to a label element does seem very strange, since the label isn't the control itself, but I doubt that's the cause of the issue.
